management-service/database/db-migrations/000003_experiment_version.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE experiments ADD version integer NOT NULL DEFAULT 1;
```

Comment:
The default value automatically takes care of those experiments that have no history.